src/test/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_SwitchResponsibleOfficerAPITest.java

```diff
@@ -0,0 +1,110 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.module.mockmvc.RestAssuredMockMvc;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.justice.digital.delius.controller.advice.SecureControllerAdvice;
+import uk.gov.justice.digital.delius.data.api.ResponsibleOfficerSwitch;
+import uk.gov.justice.digital.delius.helpers.CurrentUserSupplier;
+import uk.gov.justice.digital.delius.service.AlfrescoService;
+import uk.gov.justice.digital.delius.service.ContactService;
+import uk.gov.justice.digital.delius.service.ConvictionService;
+import uk.gov.justice.digital.delius.service.DocumentService;
+import uk.gov.justice.digital.delius.service.NsiService;
+import uk.gov.justice.digital.delius.service.OffenderManagerService;
+import uk.gov.justice.digital.delius.service.OffenderService;
+import uk.gov.justice.digital.delius.service.SentenceService;
+import uk.gov.justice.digital.delius.service.UserService;
+
+import static io.restassured.config.EncoderConfig.encoderConfig;
+import static io.restassured.module.mockmvc.RestAssuredMockMvc.given;
+import static io.restassured.module.mockmvc.config.RestAssuredMockMvcConfig.newConfig;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@ExtendWith(MockitoExtension.class)
+public class OffendersResource_SwitchResponsibleOfficerAPITest {
```

Comment:
Is it worth adding a 401 and 403 test now so it's not forgotten later?

Reply:
Also maybe something for the DELETEs?

Reply:
I was going to add those when I do an integration test, which will be first task